tools/telemetry-generator/src/handlers/testPassRate.ts

```diff
@@ -10,23 +10,23 @@
  * @param fileData - A JSON object obtained by calling JSON.parse() on the contents of a file.
  * @param logger - An ITelemetryBufferedLogger. Call its send() method to write the output telemetry events.
  */
-module.exports = function handler(fileData, logger) {
-	if (fileData.resultSummary == null) {
+module.exports = function handler(fileData, logger): void {
+	if (fileData.resultSummary === undefined) {
```

Comment:
The external API actually returns null here, so I think we need to maintain that in the checks. In theory we don't run this handler before the results are in, but I'd say better be safe.

![image](https://github.com/microsoft/FluidFramework/assets/716334/2893d726-bbf7-4942-ab36-209bad859784)

```suggestion
	// eslint-disable-next-line @rushstack/no-new-null -- External API can return an actual null
	if (fileData.resultSummary === null || fileData.resultSummary === undefined) {
```

Reply:
Oof. That was an auto-fix change... Good catch!